internal/exec/help.go

```diff
@@ -20,43 +19,59 @@ func processHelp(
-				"the '--skip-init' flag like so 'atmos terraform <command> <component> -s <stack> --skip-init")
-			u.PrintMessage(" - 'atmos terraform deploy' command executes 'terraform apply -auto-approve' (sets the '-auto-approve' flag when running 'terraform apply')")
-			u.PrintMessage(" - 'atmos terraform deploy' command supports '--deploy-run-init=true/false' flag to enable/disable running 'terraform init' " +
-				"before executing the command")
-			u.PrintMessage(" - 'atmos terraform apply' and 'atmos terraform deploy' commands support '--from-plan' flag. If the flag is specified, " +
-				"the commands will use the planfile previously generated by 'atmos terraform plan' command instead of generating a new planfile")
-			u.PrintMessage(" - 'atmos terraform apply' and 'atmos terraform deploy' commands commands support '--planfile' flag to specify the path " +
-				"to a planfile. The '--planfile' flag should be used instead of the planfile argument in the native 'terraform apply <planfile>' command")
-			u.PrintMessage(" - 'atmos terraform clean' command deletes the '.terraform' folder, '.terraform.lock.hcl' lock file, " +
-				"and the previously generated 'planfile', 'varfile', and 'backend.tf.json' file for the specified component and stack. " +
-				"Use the --everything flag to also delete the Terraform state files and directories for the component. " +
-				"Note: State files store the local state of your infrastructure and should be handled with care, if not using a remote backend.\n\n" +
-				"Additional flags:\n" +
-				"  --force Forcefully delete Terraform state files and directories without interaction\n" +
-				"  --skip-lock-file Skip deleting the '.terraform.lock.hcl' file\n\n" +
-				"If no component or stack is specified, the clean operation will apply globally to all components.")
-			u.PrintMessage(" - 'atmos terraform workspace' command first runs 'terraform init -reconfigure', then 'terraform workspace select', " +
-				"and if the workspace was not created before, it then runs 'terraform workspace new'")
-			u.PrintMessage(" - 'atmos terraform import' command searches for 'region' in the variables for the specified component and stack, " +
-				"and if it finds it, sets 'AWS_REGION=<region>' ENV var before executing the command")
-			u.PrintMessage(" - 'atmos terraform generate backend' command generates a backend config file for an 'atmos' component in a stack")
-			u.PrintMessage(" - 'atmos terraform generate backends' command generates backend config files for all 'atmos' components in all stacks")
-			u.PrintMessage(" - 'atmos terraform generate varfile' command generates a varfile for an 'atmos' component in a stack")
-			u.PrintMessage(" - 'atmos terraform generate varfiles' command generates varfiles for all 'atmos' components in all stacks")
-			u.PrintMessage(" - 'atmos terraform shell' command configures an environment for an 'atmos' component in a stack and starts a new shell " +
```

Comment:
We haven't documented `shell` subcommand. The command itself is documented, but not in the `atmos terraform --help` menu.